src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -247,7 +264,6 @@ const StudentApplication = ({ userProfile, authentication }) => {
             ' because they are not a student.',
         );
         setSnackbarSeverity('warning');
-        setSnackbarOpen(true);
       } else if (newApplicantProfile.Gender && newApplicantProfile.Gender !== userProfile.Gender) {
```

Comment:
Did you mean to repeat the same value (newApplicantProfile.Gender)?

Reply:
It is a common reliability practice that I see a lot in React code or JS in general, where you use `if (variable && variable === something)` rather than just `if (variable === something)`. This is done to explicitly check whether the thing happens to be null or false or some other invalid state before trying to make the comparison.

Reply:
Okay, interesting. Good thinking, then.